src/components/pages/group/form/speciesCheckBox/adminInviteField/index.tsx

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import SelectAsync from "@components/form/select-async";
+import TextAreaField from "@components/form/textarea";
+import useTranslation from "@configs/i18n/useTranslation";
+import { axUserSearch } from "@services/auth.service";
+import { FormContextValues } from "react-hook-form";
+
+interface AdminListDeatils {
+  adminTitle: string;
+  adminList: any[];
+  form: FormContextValues<any>;
+}
+
+interface AdminDeatils {
+  name: string;
+  label?: string;
+  textBoxName: string;
+  textBoxLabel?: string;
+  fieldGroupTitle: string;
+  isMultiple?: boolean;
+}
+
+const AdminInviteField = (props: AdminListDeatils) => {
+  const { t } = useTranslation();
+  const onAdminUserQuery = async (q) => {
```

Comment:
Please format code properly leave space after each function for better readability